src/Solution/ParticleTracker/MethodCellTernary.f90

```diff
@@ -17,13 +18,13 @@ module MethodCellTernaryModule
 
   type, extends(MethodType) :: MethodCellTernaryType
     private
-    double precision :: x_vert(99), y_vert(99) !< cell vertex coordinates (kluge 99, todo: allocate as needed)
-    double precision :: xctr, yctr !< cell center coordinates
-    double precision :: vx_vert_polygon(99), vy_vert_polygon(99) !< cell vertex velocities
-    double precision :: vxctr, vyctr !< cell center velocities
-    double precision :: ztop, zbot !< cell top and bottom elevations
-    double precision :: dz !< cell thickness
-    double precision :: vztop, vzbot !< cell top and bottom velocities
+    real(DP), allocatable :: x_vert(:), y_vert(:) !< cell vertex coordinates
+    real(DP), allocatable :: vx_vert_polygon(:), vy_vert_polygon(:) !< cell vertex velocities
```

Comment:
style-wise I think we want variables on their own lines.

Reply:
ok. I will go back and update here and elsewhere